docs/content/preview/yugabyte-voyager/reference/diagnostics-report.md

```diff
@@ -43,7 +43,7 @@ The payload for each migration phase is described in the following table. Note t
 
 | Migration phase | Data collected |
 | :---- | :------------- |
-| <br/>[assess-migration](../assess-migration/) |<ul><li> Unsupported features (reported in the assessment report without any object names or SQL statements; only count of object per feature)</li> <li>Unsupported datatypes (reported in the assessment report without any table name or column name; only the datatype)</li> <li>Error message while running the assessment if any </li> <li>Table sizing statistics (reads/writes per second, and size in bytes)</li> <li>Index sizing statistics (reads/writes per second, and size in bytes)</li> <li>Schema summary (reported in the assessment report, without object names for each object type)</li> <li>Source connectivity (whether assessment is run with source connectivity) </li></ul>|
+| <br/>[assess-migration](../assess-migration/) |<ul><li> Unsupported features (reported in the assessment report without any object names or SQL statements; only count of object per feature)</li> <li>Unsupported datatypes (reported in the assessment report without any table name or column name; only the datatype)</li> <li>Error message while running the assessment if any </li> <li>Table sizing statistics (reads/writes per second, and size in bytes)</li> <li>Index sizing statistics (reads/writes per second, and size in bytes)</li> <li>Schema summary (reported in the assessment report, without object names for each object type)</li> <li>Source connectivity (whether assessment is run with source connectivity) </li> <li>Unsupported query constructs (reported in the assessment report, without any queries information; only count of queries per construct)</li><li>Migration caveats (reported in the assessment report, without any object name or SQL statements; only count of object per caveat) </li></ul>|
```

Comment:
Migration caveates appears to come before unsupported_query_constructs in the report example; what is the order of this list based on

Reply:
changing the order based on the example. @priyanshi-yb - let me know whats the right order.